pkg/ccl/logictestccl/tests/local/BUILD.bazel

```diff
@@ -12,7 +12,7 @@ go_test(
         "//c-deps:libgeos",  # keep
         "//pkg/ccl/logictestccl:testdata",  # keep
     ],
-    shard_count = 19,
```

Comment:
Mistake?

Reply:
i suppose so. i must have something saved locally that caused this when i did `./dev gen`